src/Core/src/Handlers/IViewHandler.cs

```diff
@@ -9,7 +20,8 @@ public interface IViewHandler
 		void SetVirtualView(IView view);
 		void UpdateValue(string property);
 		void DisconnectHandler();
-		object? NativeView { get; }
+		NativeView? View { get; }
```

Comment:
Everywhere we are using this we are casting it to the NativeView
If we want to keep this as object then maybe we can add a generic interface.